test/LRPTests/pch.cpp

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
```

Comment:
Copyright should be:
// Copyright (c) Microsoft Corporation and Contributors.
// Licensed under the MIT License.